applications/luci-app-sqm/root/usr/share/luci/menu.d/luci-app-sqm.json

```diff
@@ -0,0 +1,13 @@
+{
+	"admin/network/sqm2": {
```

Comment:
nit: why sqm2?

Reply:
(while I was testing this I renamed it to sqm2 to have both the old and the new implementation) will fix.
Any other comment?

Reply:
Thanks! And no, not yet, will need to take it for a spin first...